android/runtime/v8/src/native/TypeConverter.cpp

```diff
@@ -663,20 +663,18 @@ jobject TypeConverter::jsValueToJavaObject(v8::Isolate* isolate, JNIEnv *env, v8
 
 			for (int i = 0; i < numKeys; i++) {
```

Comment:
Again, this change at the heart of our runtime scares me, but I think since KrollDict extends HashMap\<String, Object\> that always generating a KrollDict would be ok...

And on a real tangent, I don't see why we actually use HashMap in our APIs everywhere and not just Map to be more generic.